tools/ci/appveyor/appveyor_build.sh

```diff
@@ -55,25 +55,26 @@ function set_configuration {
         mingw)
             build='--build=i686-pc-cygwin'
             host='--host=i686-w64-mingw32'
+	    dep='--disable-dependency-generation'
```

Comment:
(minor: the indentation is weird here)

Reply:
Oops, new PC, misconfigured Vim, no pre-commit hook! 😊